lerna.json

```diff
@@ -0,0 +1,16 @@
+{
+  "lerna": "2.0.0-beta.30",
```

Comment:
I think that Im using features that seems to require version 2..
Looks like the rest of the big projects that uses lerna are using beta as well